sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -842,7 +842,11 @@ object SparkSession extends api.BaseSparkSessionCompanion with Logging {
     /** @inheritdoc */
     override def enableHiveSupport(): this.type = synchronized {
       if (hiveClassesArePresent) {
+        // TODO(SPARK-50244): We now isolate artifacts added by the `ADD JAR` command. This will
+        //  break an existing Hive use case (one session adds JARs and another session uses them).
+        //  We need to decide whether/how to enable isolation for Hive.
```

Comment:
@xupefei Everyone has Hive on their classpath (because of HMS), so this effectively disables isolation for all of classic. Can we only disable this for JDBC?